### PR TITLE
Adding multiple og:image:width and :height is now possible.

### PR DESCRIPTION
**Pull request by jeroendesloovere**
Closes https://github.com/forkcms/forkcms/issues/1474

### CODE REVIEW

File: src/Frontend/Core/Engine/Header.php [hunk 1]
```diff
@@ -333,14 +341,21 @@ public function addOpenGraphData($key, $value, $overwrite = false)
...
      * @param bool   $overwrite Should we overwrite the previous value?
+     * @param integer $width    The width of the image.
+     * @param integer $height   The height of the image.
      */
-    public function addOpenGraphImage($image, $overwrite = false)
+    public function addOpenGraphImage($image, $overwrite = false, $width = 0, $height = 0)
     {
+        // recast width and height
+        $width = (int) $width;
+        $height = (int) $height;
```
Comment (WouterSioen) [inline src/Frontend/Core/Engine/Header.php]: I'd throw an exception if you add both height and width, since you only use one of then.

Comment (WouterSioen) [inline src/Frontend/Core/Engine/Header.php]: Hmm, if you have both width and height, you add the meta data twice? Is this correct?

Comment (jeroendesloovere) [inline src/Frontend/Core/Engine/Header.php]: Yes, doing something like:
```php
$this->header->addOpenGraphImage('my-image-01.jpg', false, 1600, 1200);
```
Will result in
```html
<meta property="og:image" content="my-image-01.jpg">
<meta property="og:image:width" content="1600">
<meta property="og:image:height" content="1200">
```

**Comment by WouterSioen** [comment]
oops, sorry

**Pull request merged**